bundles/net.tourbook/src/net/tourbook/ui/views/tagging/TaggingView.java

```diff
@@ -1420,6 +1421,31 @@ public void update(final ViewerCell cell) {
       });
    }
 
+   private void defineColumn_Tour_TagImageFilePath() {
+
+      final TreeColumnDefinition colDef = TreeColumnFactory.TOUR_TAG_IMAGE_FILE_PATH
+            .createColumn(_columnManager, _pc);
+
+      colDef.setLabelProvider(new CellLabelProvider() {
+
+         @Override
+         public void update(final ViewerCell cell) {
+
+            cell.setText(UI.EMPTY_STRING);
+
+            final Object element = cell.getElement();
+            if (element instanceof TVITagView_Tag) {
+
+               final TVITagView_Tag tagItem = (TVITagView_Tag) element;
+
+               cell.setText(TourDatabase.getTagPropertyValue(
+                     (tagItem).tagId,
+                     "TOUR_TAG_IMAGE_FILE_PATH")); //$NON-NLS-1$
```

Comment:
This string `"TOUR_TAG_IMAGE_FILE_PATH"` is used in several files and should be replaced with a java "constant", the original code with `"TOUR_TAG_AND_CATEGORY_NOTES"` was not better

Reply:
Fixed. I used the same approach used in [TableColumnFactory](https://github.com/mytourbook/mytourbook/blob/6fa76700a5a69b11d80a99f9b8d1106acfbb37f7/bundles/net.tourbook/src/net/tourbook/ui/TableColumnFactory.java#L43) and used a Constant ID. I did the same for `"TOUR_TAG_AND_CATEGORY_NOTES"`.
But as I didn't see any ID created in `TreeColumnFactory`, let me know if it's not a good solution.